WordPress/Classes/ViewRelated/Comments/CommentDetailViewController.swift

```diff
@@ -181,9 +181,22 @@ private extension CommentDetailViewController {
     }
 
     func configureNavigationBar() {
-        if comment.canModerate {
-            navigationItem.rightBarButtonItem = UIBarButtonItem(barButtonSystemItem: .edit, target: self, action: #selector(editButtonTapped))
-        }
+        // apply a Messages-like visual blur effect when content is scrolled.
```

Comment:
Nitpick: the `Messages-like` description will be misleading when Messages changes again. Can we just remove that bit?

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17255/commits/ffc524b622932fb3972cc20b0158697dac0a89f7. Thank you